templates/media/media--image.html.twig

```diff
@@ -0,0 +1,41 @@
+{#
+/**
+ * @file
+ * Theme override to display an image media item with WA-compliant description.
+ * ~ When field_aria_description has value, this template:
+ * ~~ Adds aria-describedby="description-[MID]" to field_media_image <img> tag.
+ * ~~ Renders field_aria_description inside div#description-[MID].sr-only.
+ * See cwd_base/media/media.html.twig for available variables and other info.
+ *
+ * @see template_preprocess_media()
+ *
+ * @ingroup themeable
+ */
+#}
+{{ title_suffix.contextual_links }}
+{% if content %}
+
+  {% if media.field_aria_description.0.value -%}
+    {% if media.field_media_image.0.value -%}
+
+      {####### variable setup #######}
+      {% set describedby = 'description-' ~ media.mid.0.value %}
+      {% set compleximg_alt = media.field_media_image[0].alt ?? '' %}
+      {% set compleximg_url = media|file_uri %}
+      {% set style_name = content.field_media_image[0]['#image_style']|default('') %}
+
+      {####### output #######}
+      {{ drupal_image(compleximg_url, style_name, {alt: compleximg_alt, 'aria-describedby': describedby} }}
```

Comment:
Another typo haha -- note to self, copy code from suggested changes into normal text editor before committing, to check for typos.  A bummer, cuz this suggested changes thing is extremely cool, but, not quite a full text editor :grin: 
```suggestion
      {{ drupal_image(compleximg_url, style_name, {alt: compleximg_alt, 'aria-describedby': describedby}) }}
```